src/System.Console/tests/System.Console.Tests.csproj

```diff
@@ -36,6 +36,11 @@
     </Compile>
     <Compile Include="WindowAndCursorProps.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <SupplementalTestData Include="$(MSBuildThisFileDirectory)TestData\**\*.*">
+      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </SupplementalTestData>
+  </ItemGroup>
```

Comment:
Historically we've put binary files into the testdata repo to avoid bloating the corefx repo (once files are added, they're in the history forever). How big are these? Should they be added to the testdata repo instead?

Reply:
I talked to @weshaggard about this, and he said that if the files where no bigger than a MB then it makes more sense to just check them in along with the test project, and these guys are about 4KB each.